src/ring_theory/ideal/basic.lean

```diff
@@ -79,6 +79,12 @@ theorem eq_top_iff_one : I = ⊤ ↔ (1:α) ∈ I :=
 theorem ne_top_iff_one : I ≠ ⊤ ↔ (1:α) ∉ I :=
 not_congr I.eq_top_iff_one
 
+lemma exists_mem_ne_zero_of_ne_bot (hI : I ≠ ⊥) : ∃ p ∈ I, p ≠ (0 : α) :=
```

Comment:
Would you mind adding and `iff` variant?

It can probably be proved using
```
have := eq_bot_iff, rw <- not_iff_not at this, push_neg at this, exact this
```
or something like that.